simul/platform/MININET.md

```diff
@@ -98,26 +109,45 @@ which should remove all running mininet sessions and return to the command line.
 ### Monitor port occupied
 
 Sometimes the ssh-forwarding is misbehaving, and you cannot run your simulation.
-The simplest solution is to restart the servers (not reinstall, just restart).
+The simplest solution is to restart the servers (not reinstall, just power cycle it).
 A faster option is to ssh to the first server and check if you can find the rogue
-`sshd` process. Take care, there is a `sshd`-process that listens for incoming
+`sshd` process.
+
+Take care, there is a `sshd`-process that listens for incoming
 connections - if you kill that one, you will have to restart the server, as you
-won't have access to it...
+won't have access to it.
```

Comment:
You could add the 

```
pkill -9 -f "ssh -nNTf"
```

command here from https://github.com/dedis/onet/blob/c2ba2e8b58f4d48f7a11a1e6776c4c02d5b980a6/simul/platform/mininet.go#L202 - is that still correct?

Reply:
I haven't needed anything like that, so no?